uui/settings.ts

```diff
@@ -1,315 +1,1148 @@
+    defaultFooterVPadding: FlexRowProps['vPadding'];
+}
+
+interface ModalSettings {
+    icons: ModalIcons;
+    sizes: ModalSizes;
+}
+
+const modalSettings: ModalSettings = {
+    icons: {
+        closeIcon: CrossIcon,
+    },
+    sizes: {
+        defaultWidth: 420,
+        defaultMaxHeight: '80dvh',
+        defaultMobileMaxHeight: '100dvh',
+        defaultHeaderPadding: '24',
+        defaultFooterColumnGap: '12',
+        defaultFooterPadding: '24',
+        defaultFooterVPadding: '24',
+    },
+};
+
+interface NotificationCardIcons {
+    closeIcon: Icon;
+    hintIcon: Icon;
+    successIcon: Icon;
+    warningIcon: Icon;
+    errorIcon: Icon;
+}
+
+interface NotificationCardSizes {
+    action: LinkButtonProps['size'];
+}
+
+interface NotificationCardSettings {
+    icons: NotificationCardIcons;
+    sizes: NotificationCardSizes;
+}
+
+const notificationCardSettings: NotificationCardSettings = {
+    icons: {
+        closeIcon: CrossIcon,
+        hintIcon: HintIcon,
+        successIcon: SuccessIcon,
+        warningIcon: WarningIcon,
+        errorIcon: ErrorIcon,
+    },
+    sizes: {
+        action: '30',
+    },
+};
+
+interface NumericInputIcons {
+    arrowIcon: Icon;
+}
+
+interface NumericInputSizes {
+    default: NumericInputProps['size'];
+}
+
+interface NumericInputSettings {
+    icons: NumericInputIcons;
+    sizes: NumericInputSizes;
+}
+
+const numericInputSettings: NumericInputSettings = {
+    icons: {
+        arrowIcon: DropdownIcon,
+    },
+    sizes: {
+        default: '36',
+    },
+};
+
+interface PaginatorIcons {
+    leftArrowIcon: Icon;
+    rightArrowIcon: Icon;
+}
+
+interface PaginatorSizes {
+    default: ButtonProps['size'];
+}
+
+interface PaginatorSettings {
+    icons: PaginatorIcons;
+    sizes: PaginatorSizes;
+}
+
+const paginatorSettings: PaginatorSettings = {
+    icons: {
+        leftArrowIcon: LeftArrowIcon,
+        rightArrowIcon: RightArrowIcon,
+    },
+    sizes: {
+        default: '36',
+    },
+};
+
+interface PickerInputIcons {
+    toggler: {
+        clearIcon: Icon;
+        dropdownIcon: Icon;
+    },
+    body: {
+        selectIcon: Icon;
+        boldSelectIcon: Icon;
+        pickerBodyMobileHeaderCloseIcon: Icon;
+        modalNotFoundSearchIcon: Icon;
+    }
+}
+
+interface IsBoldSizes {
+    [size: string | number]: boolean;
+}
+
+interface PickerInputSizes {
+    toggler: {
+        default: PickerTogglerProps<unknown, unknown>['size'];
+        defaultTag: PickerTogglerTagProps<unknown, unknown>['size'];
+        tagMap: Sizes<PickerTogglerTagProps<unknown, unknown>['size'], TagProps['size']>;
+    };
+    body: {
+        maxHeight: number;
+        minWidth: number;
+        padding: DataPickerRowProps<unknown, unknown>['padding'];
+        defaultRow: DataPickerCellProps<unknown, unknown>['size'];
+        defaultItem: PickerItemProps<unknown, unknown>['size'];
+        defaultFooter: DataPickerFooterProps<unknown, unknown>['size'];
```

Comment:
Use defaultRow value and remove:
 defaultItem: PickerItemProps<unknown, unknown>['size'];
        defaultFooter: DataPickerFooterProps<unknown, unknown>['size'];